jujugui/static/gui/src/app/components/store/_store.scss

```diff
@@ -92,41 +92,38 @@
   }
 
   .box {
+    min-height: 310px;
     margin-bottom: 20px;
+    padding-bottom: 125px;
 
     &--kubernetes {
       overflow: hidden;
-      padding-bottom: 125px;
-      background-color: #3371e3;
+      background: #3371e3 url(https://assets.ubuntu.com/v1/2847e80a-k8-image.png) no-repeat center -50px;
```

Comment:
Do we no longer need all assets local. Originally this was to support offline usage.